src/api/libcellml/import.h

```diff
@@ -36,24 +36,55 @@ class LIBCELLML_EXPORT Import: public Entity
     Import& operator=(Import m); /**< Assignment operator */
 
     /**
-     * @brief Set the source Model.
+     * @brief Set the source Model's URL.
      *
-     * Set the source Model that this import refers to.
+     * Set the source Model's URL that this import refers to.
      *
-     * @param reference The reference model.
+     * @param reference The referenced model's URL.
      */
     void setSource(const std::string &reference);
 
     /**
-     * @brief Get the source model.
+     * @brief Get the source model's URL.
      *
-     * Get the source model set in this instance.  If no source model
-     * is set then return an empty string.
+     * Get the source model's URL set in this instance. If no source model
+     * URL is set then return an empty string.
      *
      * @return The reference to the source model if set otherwise the emtpy string.
      */
     std::string getSource() const;
 
+    /**
+     * @brief Provide the model used to resolve this import.
+     *
+     * Uses the provided model to resolve this import, which should correspond
+     * to the @c source identified by this import.
+     *
+     * @param model The model to use in resolving this import.
+     */
+    void resolveImport(libcellml::ModelPtr model);
```

Comment:
I think this method should be called 'setResolvingModel'

Reply:
Why not simply `setSourceModel()` (and, likewise, `getSourceModel()` rather than `getResolvingModel()`)?